src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -116,14 +116,14 @@ const RegisterForm: React.FunctionComponent<PropsType> = ({
 
   return (
     <div className="form-container">
-      <h2>Create an account!</h2>
       <div className="secondary-action">
         Already have an account?
         <br />
         <span className="link" onClick={displayLoginForm}>
           Login
         </span>
       </div>
```

Comment:
As above, can we move this below the login form as well, and potentially put it in a `<small></small>` (or equivalent CSS)

Reply:
[NB, nit, DRY] This is reused between `src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx` and here, we should extract it into a common 'dumb' component